packages/contracts/src/contracts/current/protocol/Exchange/mixins/MMatchOrders.sol

```diff
@@ -54,4 +54,17 @@ contract MMatchOrders is
         internal
         pure
         returns (LibFillResults.MatchedFillResults memory matchedFillResults);
+
+    /// @dev Settles matched order by transferring appropriate funds between order makers, taker, and fee recipient.
+    /// @param leftOrder First matched order.
+    /// @param rightOrder Second matched order.
+    /// @param takerAddress Address that matched the orders. The taker receives the spread between orders as profit.
+    /// @param matchedFillResults Struct holding amounts to transfer between makers, taker, and fee recipients.
+    function settleMatchedOrders(
+        LibOrder.Order memory leftOrder,
+        LibOrder.Order memory rightOrder,
+        address takerAddress,
+        LibFillResults.MatchedFillResults memory matchedFillResults
+    )
+        internal;
```

Comment:
Same here. By not exposing them we simplify code and limit where it can get called from.